crowdsec-docs/docs/bouncers/nginx.mdx

```diff
@@ -79,102 +94,314 @@ If you are on a mono-machine setup, the `crowdsec-nginx-bouncer` install script
+- Remove the old bouncer:
+
+```bash
+sudo apt-get remove --purge crowdsec-nginx-bouncer
+```
+
+- Install the new bouncer:
 ```bash
-git clone https://github.com/crowdsecurity/cs-nginx-bouncer.git
+sudo apt-get update
+sudo apt-get install crowdsec-nginx-bouncer
 ```
 
-#### Installation
+- Configure the bouncer in `/etc/crowdsec/bouncers/crowdsec-nginx-bouncer.conf`
 
-##### lua-cs-bouncer
 
+## References
+
+### `API_KEY`
 ```bash
-cd ./lua-cs-bouncer/
-sudo make install
+API_KEY=<API_KEY>
 ```
 
-##### crowdsec-nginx-bouncer
+CrowdSec Local API key.
+
+Generated with [`sudo cscli bouncers add`](../cscli/cscli_bouncers_add) command.
 
-- Copy the `crowdsec-nginx-bouncer/nginx/crowdsec_nginx.conf` into `/etc/nginx/conf.d/crowdsec_nginx.conf`:
+### `API_URL`
 ```bash
-cp ./crowdsec-nginx-bouncer/nginx/crowdsec_nginx.conf /etc/nginx/conf.d/crowdsec_nginx.conf
+API_URL=http://<ip>:<port>
 ```
-- Copy the `crowdsec-nginx-bouncer/nginx/access.lua` into `/usr/local/lua/crowdec/access.lua`:
+
+CrowdSec local API URL.
+
+
+### `BOUNCING_ON_TYPE`
 ```bash
-cp ./crowdsec-nginx-bouncer/nginx/access.lua /usr/local/lua/crowdec/access.lua
+BOUNCING_ON_TYPE=all|ban|captcha
 ```
 
-Configure your API url and key in `/etc/crowdsec/bouncers/crowdsec-nginx-bouncer.conf`:
+Type of remediation we want to bounce. 
+If you choose `ban` only and receive a decision with `captcha` as remediation, the bouncer will skip the decision.
 
-```lua
-API_URL=http://127.0.0.1:8080
-API_KEY=<API KEY> --generated with `cscli bouncers add -n <bouncer_name>
-LOG_FILE=/tmp/lua_mod.log
-CACHE_EXPIRATION=1
-CACHE_SIZE=1000
+### `FALLBACK_REMEDIATION`
+```bash
+FALLBACK_REMEDIATION=ban
 ```
 
-You can now restart your nginx server:
+The fallback remediation if the bouncer receive a decision with an unknown remediation.
+
+### `MODE`
 ```bash
-systemctl restart nginx
+MODE=stream|live
 ```
 
-## Upgrade
+The bouncer mode:
+ - stream: The bouncer will pull new/old decisions from the local API every X seconds (`UPDATE_FREQUENCY` parameter).
 
-If you already have `crowdsec-nginx-bouncer` installed, please download the [latest release](https://github.com/crowdsecurity/cs-nginx-bouncer/releases) and run the following commands:
+Note: The timer that pull the local API will be triggered after the first request.
+ - live: The bouncer will query the local API for each requests (if IP is not in cache) and will store the IP in cache for X seconds (`CACHE_EXPIRATION` parameter).
 
+### `REQUEST_TIMEOUT`
 ```bash
-tar xzvf crowdsec-nginx-bouncer.tgz
-cd crowdsec-nginx-bouncer-v*/
-sudo ./upgrade.sh
-sudo systemctl restart nginx
+REQUEST_TIMEOUT=1000
 ```
 
-## Configuration
+Timeout in second for the HTTP requests done by the bouncer to query CrowdSec local API or www.google.com (for the reCAPTCHA verification).
+
+### `EXCLUDE_LOCATION`
+```bash
+EXCLUDE_LOCATION=/<path1>,/<path2>
+```
+
+The locations to exclude the boucing. It is a list of location, separated by commas.
```

Comment:
The locations to exclude while bouncing. It is a list of location, separated by commas.